.rancher-pipeline.yml

```diff
@@ -16,10 +16,13 @@ stages:
     steps:
       - applyAppConfig:
           catalogTemplate: p-rlq5p:concorde-helmcharts-mod-data-export
-          version: 0.1.10
+          version: 0.1.26
           answers:
             image.repository: docker.dev.folio.org/mod-data-export
             image.tag: concorde-latest
+            javaOptions: -XX:MaxRAMPercentage=85.0 -XX:+UseG1GC -Dbucket.name=kruthi-test
```

Comment:
@KVupp Is it ok if I will add this bucket name, or it is better to wait for the proper?

Reply:
The actual buckets have been created I believe Andrii, the story seems to be closed. Let's test that out and change it to proper names